blockdevice/blockdevice.go

```diff
@@ -36,6 +36,24 @@ type BlockDevice struct {
 	// BlockDevice if it exists
 	FSInfo FileSystemInformation
 
+	// Parent is the parent device of this blockdevice, if it exists.
```

Comment:
As part of the BlockDevice struct .. comment, it will be better to list the various examples in which block devices exist and how each of these sub files relate to the block device in co-relation.